src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8799,8 +8814,8 @@ private void ParseDeclarationModifiers(SyntaxListBuilder list)
             SyntaxKind k;
             while (IsDeclarationModifier(k = this.CurrentToken.ContextualKind) || IsAdditionalLocalFunctionModifier(k))
             {
-                // "readonly ref" is not a modifier, in our syntax.
-                if (k == SyntaxKind.ReadOnlyKeyword && this.PeekToken(1).Kind == SyntaxKind.RefKeyword)
+                // "ref readonly" is not a modifier, in our syntax.
+                if (k == SyntaxKind.RefKeyword && this.PeekToken(1).Kind == SyntaxKind.ReadOnlyKeyword)
```

Comment:
Per discussion, @VSadov will handle that change.

---
In reply to: [103036678](https://github.com/dotnet/roslyn/pull/17337#discussion_r103036678) [](ancestors = 103036678)